airflow/config/airflow.cfg

```diff
@@ -49,30 +51,68 @@ dag_concurrency = 16
 # Are DAGs paused by default at creation
 dags_are_paused_at_creation = False
 
+# When not using pools, tasks are run in the "default pool",
+# whose size is guided by this config element
+non_pooled_task_slot_count = 128
+
 # The maximum number of active DAG runs per DAG
 max_active_runs_per_dag = 16
 
 # Whether to load the examples that ship with Airflow. It's good to
 # get started, but you probably want to set this to False in a production
 # environment
-#load_examples = True
 load_examples = False
 
 # Where your Airflow plugins are stored
 plugins_folder = /usr/local/airflow/plugins
 
 # Secret key to save connection passwords in the db
-fernet_key = fKp7omMJ4QlTxfZzVBSiyXVgeCK-6epRjGgMpEIsjvs=
+fernet_key = sHwbcyd5Xd2V6gKIVMA1DC9wvvLNXt9IiJQFa7k4YOo=
```

Comment:
Agree that it should be generated automatically.  I have updated the dockerfiles.  I have also updated kubectl from v1.6.2 to v1.6.5 so that it is the same as the current version used in OpenStack-Helm AIO script.